cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -33,9 +36,10 @@ const (
 )
 
 // VMTServer has all the context and params needed to run a Scheduler
+// TODO: leaderElection is disabled now because of dependency problems.
 type VMTServer struct {
 	Port            int
-	Address         net.IP
+	Address         string
```

Comment:
This is the address for kubeturbo to listen on.  
String makes it easier to get input from command line.
I think it won't cause bugs in the program. User will easily find the problem if the input is wrong.